src/app/shared/utils/markdown.pipe.ts

```diff
@@ -0,0 +1,70 @@
+import { Inject, InjectionToken, Pipe, PipeTransform } from '@angular/core';
+import MarkdownIt from 'markdown-it';
+import * as sanitizeHtml from 'sanitize-html';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+import { environment } from '../../../environments/environment';
+
+const mathjaxLoader = async () => (await import('markdown-it-mathjax3')).default;
+type Mathjax = ReturnType<typeof mathjaxLoader>;
+const MATHJAX = new InjectionToken<Mathjax>(
+  'Lazily loaded mathjax',
+  { providedIn: 'root', factory: mathjaxLoader }
+);
+
+/**
+ * Pipe for rendering markdown and mathjax.
+ */
+@Pipe({
+  name: 'dsMarkdown'
+})
+export class MarkdownPipe implements PipeTransform {
```

Comment:
Can this pipe be used in any metadata field?  It'd be nice to have a small example in the TypeDocs of how to use this Pipe, as it's not obvious what the syntax should look like in a component template.

Reply:
I've added some extra docs here to illustrate the usage. It can be used on anything, not only metadata fields (e.g. static pages)